### ISSUE TITLE
Opening the simple item page of a withdrawn item when logged in as administrator stalls the frontend instance

### ISSUE THREAD

**Issue by MW3000** [DSpace/dspace-angular#3393]
## Describe the bug
In DSpace 8.x, opening the simple item page of a withdrawn item when logged in as administrator stalls the frontend instance for about 10 to 15 minutes, making it unresponsive, then continues to display the simple item page.

## To Reproduce
1. Login as an administrator
2. Withdraw an item
3. Go the withdrawn item's simple item page, eg. by clicking the item in the breadcrumbs, searching for a withdrawn item in administrative search or because you copied the url beforehand.
4. The whole frontend instance stalls for about 10 to 15 minutes. After that time, the simple item page is displayed.

You can easily reproduce this on the DSpace Demo Website

## Expected behavior
The simple item page should come up immediately

**Comment by Leano1998** [comment]
Hi,
I looked a bit deeper into this and figured, it might be a problem with the [cache of the authorization information](https://github.com/DSpace/dspace-angular/blob/main/src/app/item-page/alerts/item-alerts.component.ts#L77):
```ts
  showReinstateButton$(): Observable<boolean>  {
    const correction$ = this.correctionTypeDataService.findByItem(this.item.uuid, true).pipe(
      getFirstCompletedRemoteData(),
      map((correctionTypeRD: RemoteData<PaginatedList<CorrectionType>>) => correctionTypeRD.hasSucceeded ? correctionTypeRD.payload.page : []),
    );
    const isAdmin$ = this.authService.isAuthorized(FeatureID.AdministratorOf);
    return combineLatest([isAdmin$, correction$]).pipe(
      map(([isAdmin, correction]) => {
        return !isAdmin && correction.some((correctionType) => correctionType.topic === REQUEST_REINSTATE);
      },
      ));
  }
```
If I force the authService not to use the cached version, it works for me, otherwise it seems to return `undefined`.
```ts
    const isAdmin$ = this.authService.isAuthorized(FeatureID.AdministratorOf, undefined, undefined, false);
```

**Comment by alexandrevryghem** [comment]
@Leano1998: The problem is most likely that the template always recreates a new `Observable` every time the template is rerenderd. I already have a branch with similar fixes on other places that I was going to contribute, I can add that fix to that branch too 🤷 
https://github.com/DSpace/dspace-angular/blob/8cc45e8cbed8b6d1f101c3b15be1de63ba1ed083/src/app/item-page/alerts/item-alerts.component.html#L11

**Comment by Leano1998** [comment]
@alexandrevryghem : Yes, I noticed this behavior in the browser console. Thank you for the adding the fix!

**Comment by alanorth** [comment]
I haven't noticed this on DSpace 7.6.x. We have many withdrawn items so I would have definitely run into this.

**Comment by alexandrevryghem** [comment]
Indeed, only `dspace-8_x` and `main` are affected since this originates from [this PR](https://github.com/DSpace/dspace-angular/pull/2759), which hasn't been backported to `dspace-7_x`.

**Comment by alanorth** [comment]
I know that I said 7.6.x wasn't affected, but a comment on Slack regarding thumbnails reminded me that we do seem to have some glitch with withdrawn items as well. See this video:

[Screencast From 2024-11-04 15-58-11.webm](https://github.com/user-attachments/assets/9c3a8a0a-99ce-4710-9ffd-467aa0b3adf9)

This item has a thumbnail in its `THUMBNAIL` bundle and I am logged in as an administrator. I see the spinning indicator for the thumbnail, and in the console I see an HTTP 401 for the content. Interestingly, when I move the cursor over the "edit item" button the placeholder appears. Sometimes I see the thumbnail appear, though I can't figure out the sequence of events to reproduce it.

**Comment by alexandrevryghem** [comment]
@alanorth: This makes me think of an old fix that I made once but haven't contributed yet, could you try testing if [this fix](https://github.com/alexandrevryghem/dspace-angular/commit/cdb58f2875ca9d891a112e12fcc553105c9782f6) fixes your issue. It has been in my backlog for a while so I'm not 100% sure if it's even necessary anymore 😅

**Comment by saschaszott** [comment]
This bug could be related to the bug described in issue https://github.com/DSpace/dspace-angular/issues/3584.

**Comment by MW3000** [comment]
@alexandrevryghem It might be related to #3584. But I just read above, it is marked as completed by #3585, which is not ported yet to DSpace 8?

> [tdonohue](https://github.com/tdonohue) closed this as [completed](https://github.com/DSpace/dspace-angular/issues?q=is%3Aissue+is%3Aclosed+archived%3Afalse+reason%3Acompleted) in https://github.com/DSpace/dspace-angular/pull/3585 [2 weeks ago]

As the bugreport was for DSpace 8 and our users are still experiencing the bug, I thought it would be good to have the issue open until fixed in DSpace 8?

**Comment by tdonohue** [comment]
@MW3000 : Closing this again as it's *fixed* in the codebase (for 9.0)... once a bug is fixed, the linked bug ticket will auto-close as this is an indication to developers that it's been verified to be fixed (but it still may not yet be released, obviously).

However, I've added a reminder to @alexandrevryghem on #3585 that we need to backport it to both `dspace-8_x` (and `dspace-7_x` if applicable).   So, this bug will be fixed in 8.x as well, but it still requires (manual) backporting.

@alexandrevryghem if you can find time to backport this sooner (rather than later), I'd appreciate it.  I can gladly review, and perhaps @MW3000 can help us to test the backport.  We need to include this fix in 8.1.

**Comment by MW3000** [comment]
@tdonohue I did test the port to 8.x https://github.com/DSpace/dspace-angular/pull/3738 and left a comment.
